docs/checkers/index.md

```diff
@@ -2557,6 +2557,16 @@ and `pdb.set_trace()`) are found. These breakpoints should be removed in product
 
 ```
 
+## Modified Iteration
+
+### Not a modifiable dictionary
+
+This error occurs when a dictionary is modified while it is in a for loop. This modification includes adding or removing items from the `dict`.The resulting message is, Iterated dict '%s' is being modified inside for loop body, iterate through a copy of it instead.
```

Comment:
For the first part, you can combine the two sentences by saying "...is modified inside a for loop by adding or removing items." Other types of modification (like assigning a new value to an existing key) are actually okay, and do not trigger the error.

For the second part, you don't need to say what the error message is. Instead, use plain English to describe the alternative (iterating through a copy of the dictionary); remember, you are writing for beginners to understand here!

Also, make sure there's a space after every `.`